lib/index.js

```diff
@@ -0,0 +1,147 @@
+"use strict";
```

Comment:
This file is temporarily checked in while we test this repo in terra-core/terra-framework from the GitHub reference.